Library/ViewModels/RemoteConfigFeatureFlagToolsViewModel.swift

```diff
@@ -38,7 +38,7 @@ public final class RemoteConfigFeatureFlagToolsViewModel: RemoteConfigFeatureFla
     let remoteConfigFeatures = features
       .map { features in
         features.map { feature -> (RemoteConfigFeature, Bool) in
-          (feature, AppEnvironment.current.remoteConfigClient?.isFeatureEnabled(featureKey: feature) ?? false)
+          (feature, isFeatureEnabled(feature))
```

Comment:
needed to change this back to using the helper methods directly since RemoteConfigClientType uses the RemoteConfig instance to get values instead of userDefaults, which is what we need to use in our local flags

Reply:
Good catch!